### DISCUSSION TITLE
How can I edit anything in the login screen?

### DISCUSSION THREAD

**Discussion by guiqis** [chatwoot/chatwoot#8975] [Self hosted]
I'm trying (without success) to edit anything in the login screen. I've tried to add console logs, change texts and stuff, but none of my changes appear to be applied. I did recompile, cleared cache, event restarted the server.

How can I edit something and see changes being applied?

As a reference, I'm editing `app/javascript/v3/views/login/Index.vue`

**Comment by jordan26** [comment]
I too want to make some admin page edits, such as the sidebar for us is too thin, it's cutting off half the inbox names. I want to make changes to the CSS etc too but not sure how to re-compile. @guiqis would you mind providing instruction how you are re-compiling? Even if it doesn't work yet, I'll begin from there and test too, if/when I figure anything out I'll report back too.

**Reply by jordan26** [reply]
Update, just found this thead - https://github.com/orgs/chatwoot/discussions/8899

**Reply by guiqis** [reply]
I saw this one, but my problem is not on CSS. The weird is that, when I changed the layout inside `app/view/layouts` the changes apply. But in the login page, nothing happens.

I'm recompiling using the guide available in the update instructions for Linux VM (https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm#upgrading-to-a-newer-version-of-chatwoot)
